pyscript.core/tests/integration/test_02_display.py

```diff
@@ -342,9 +353,7 @@ def test_display_HTML(self):
         assert out.inner_html() == "<p>hello world</p>"
         assert out.inner_text() == "hello world"
 
-    # waiit_for_pyscript is broken: it waits until the python code is about to
-    # start, to until the python code has finished execution
-    @pytest.mark.skip("FIXME: wait_for_pyscript is broken")
+    @skip_worker("NEXT: matplotlib-pyodide backend does not work")
```

Comment:
this is cute but it feels like, while we do care about matplotlib working from workers, and it's tested and it does work already, we're testing 3rd party gotchas in our unrelated project. This should (imho) be part of examples tests, not core functionalities.